client/src/components/util/truncate.js

```diff
@@ -40,7 +40,7 @@ export default (props) => {
   ) {
     throw Error("Only pass a single child with text to Truncate");
   }
-  const originalString = children.props.children;
+  const originalString = children.props.children.toString();
```

Comment:
suggest `String(children.props.children)` over `toString()`, since:

`String(undefined) // undefined` while `undefined.toString()` blows up the program and throws a runtime error!

Reply:
Nice catch but shouldn't
```javascript
  if (
    React.Children.count(children) !== 1 ||
    React.Children.count(children.props?.children) !== 1
  ) {
    throw Error("Only pass a single child with text to Truncate");
  }
```
catch that before it can error?

Reply:
Yes! But for consistency, personally I find it easier to always use `String()` and haven't found a need to use `toString()` 😆

Reply:
Oh! Just double checking, did we need to use `toString()` here or it's just not updated yet 😆 ? Thank you!